### ISSUE TITLE
Can you clarify pigmentary retinopathy versus retinitis pigmentosa?

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9621]
**HPO term**
HP:0000580 and HP:0000510

**Suggested revision of textual definition**
HP:0000510 is under retinal dystrophy, and HP:0000580 is under abnormality of retinal pigmentation, but they describe very related concepts.  Can the relationship be better defined?

**Comment by pnrobinson** [comment]
@sukram @psergouniotis 
We have used "Retinitis pigmentosa" as an HPO term because this is what is used in the medical genetics literature for diseases such as Bardet Biedl syndrome that include retinitis pigmentosa-like findings.
At some point we renamed the term to
[Rod-cone dystrophy HP:0000510](https://hpo.jax.org/app/browse/term/HP:0000510) and kept "Retinitis pigmentosa" as a synonym. 
Possibly it would be better to merge this with [Pigmentary retinopathy HP:0000580](https://hpo.jax.org/app/browse/term/HP:0000580) which is less of a "disease-like" label and I think it is basically a synonym? 

Thoughts?

**Comment by pnrobinson** [comment]
@daboe01, Amélie 
Would it be good to retain 
[Pigmentary retinopathy HP:0000580](https://hpo.jax.org/browse/term/HP:0000580)

and to merge [Rod-cone dystrophy HP:0000510](https://hpo.jax.org/browse/term/HP:0000510) into it? 

Pigmentary retinopathy would thus be one of the phenotypic features of the disease Rod-code dystrophy. Historically, many of the OMIM entries just use "Retinitis pigmentosa" as a description for diseases such as Bardet Biedl syndrome.
